docs/gallery/plot_external_resources.py

```diff
@@ -320,3 +323,40 @@ def __init__(self, **kwargs):
 
 er_read = ExternalResources.from_norm_tsv(path='./')
 os.remove('./er.zip')
+
+###############################################################################
+# Using TermSet with ExternalResources
+# ------------------------------------------------
+# :py:class:`~hdmf.TermSet` allows for an easier way to add references to
+# :py:class:`~hdmf.common.resources.ExternalResources`. These enumerations take place of the
+# entity_id and entity_uri parameters. :py:class:`~hdmf.common.resources.Key` values will have
+# to match the name of the term in the :py:class:`~hdmf.TermSet`.
+try:
+    import linkml_runtime  # noqa: F401
+    LINKML_INSTALLED = True
+except ImportError:
+    LINKML_INSTALLED = False
+
+if LINKML_INSTALLED:
+    from hdmf.term_set import TermSet
```

Comment:
```suggestion
import linkml_runtime
from hdmf.term_set import TermSet
```

Can't we just assume that linkml is installed for these gallery examples? If we put linkml in `requirements-doc.txt` then it will be available on readthedocs. It will also be available when devs build the documentation because they are supposed to install deps from `requirements-doc.txt`. Note that `requirements-doc.txt` contains other deps that are normally not installed like matplotlib.